From 0e9a2e346fa78952cc99011443561f210151c81e Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Mon, 6 Feb 2023 10:14:30 +0800
Subject: [PATCH 34/45] Revert "drm/tegra: Add missing clk_disable_unprepare()
 in tegra_dc_probe()"

This reverts commit cf2cbca714725d2a7698e777a4263b4f33483ac6.
---
 drivers/gpu/drm/tegra/dc.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/tegra/dc.c b/drivers/gpu/drm/tegra/dc.c
index 958d12da902d3..ceb86338c0039 100644
--- a/drivers/gpu/drm/tegra/dc.c
+++ b/drivers/gpu/drm/tegra/dc.c
@@ -2564,10 +2564,8 @@ static int tegra_dc_probe(struct platform_device *pdev)
 	usleep_range(2000, 4000);
 
 	err = reset_control_assert(dc->rst);
-	if (err < 0) {
-		clk_disable_unprepare(dc->clk);
+	if (err < 0)
 		return err;
-	}
 
 	usleep_range(2000, 4000);
 
-- 
2.25.1

